java/src/main/java/org/whispersystems/signalservice/internal/websocket/WebSocketConnection.java

```diff
@@ -42,7 +43,7 @@
 public class WebSocketConnection extends WebSocketListener {
 
   private static final String TAG                       = WebSocketConnection.class.getSimpleName();
-  private static final int    KEEPALIVE_TIMEOUT_SECONDS = 55;
+  private static final int    KEEPALIVE_TIMEOUT_SECONDS = 60;
```

Comment:
Please revert this

Reply:
As far as I recall, alarms cannot be set less than a minute apart, but perhaps that was the case only for repeating alarms.  If that is the case though and if the 55 second value is critical, say because the timeout is set to exactly 60 seconds on the server side, it will lead to intermittent EOF exceptions in the non-GCM case, unless the interval is raised a bit on the server side.  I'll have to re-check when I get some time.

Reply:
It seems that non-repeated alarms can indeed be repeatedly set, spaced less than a minute apart, so I reverted the change.  I hope this holds on all devices, as the documentation doesn't state this clearly.